docs/modules/install/pages/customization.adoc

```diff
@@ -0,0 +1,85 @@
+# Change logic in decdidim-core #
+
+There are occasions you find yourself in a situation that you need to change Decidim core logic.  There are two ways to accomplish this goal, and we are going through these options here.
+
+## 1. Direct change from your application ##
+
+This method is more straight-forward, and we discuss it here first. To change the logic in decidim, identify which part of the source code you need to change. The best way to achieve this is to identify the module being responsible for handling the task. Next, create the same path on our project, and copy the file to the same destination in our project. Finally, change the code as you wish and make sure that the changes are being applied.
+Lets have an example to make this clear. Lets add a rule to the Admin panel, where Admin can not add projects with budgets more than half the total budget. Once we applied such limitation, admin should not be able to add a proposal with the cost of 5,001 Euros to a budget with total of 10,000 Euros.
+
+We want to add a validation rule to achieve this goal. For that to happen, we go through the following steps:
+
+. First, we need to investigate the original source code, where the validation for new projects being generated(in this case, it is decidim-budgets/App/form/admin/create_proposal).
+. Now that we find the original source code, we need to over-ride this file. For that to happen, we create the following destination(if it has not already existed), and copy all contents of the original files there(in this particular example, change the validation for :budget_amount.
+
+NOTE: Please note the version of decidim on github, since it may not be in sync with your local version.
```

Comment:
Please remove this. I think we should not be referring to GitHub here because it can be confusing if there are version mis-matches within the copied files. Let's suggest to copy it directly from the correct gem version as suggested above.